doc/sphinx/source/releasenotes.rst

```diff
@@ -10,45 +10,70 @@ for each release of libCEED.
+* New Python interface using `CFFI <https://cffi.readthedocs.io/>`_ provides a nearly
+  1-1 correspondence with the C interface, plus some convenience features.  For instance,
+  data stored in the :cpp:type:`CeedVector` structure are available without copy as
+  :py:class:`numpy.ndarray`.
+* Linear QFunctions can be assembled as block-diagonal matrices (per quadrature point,
+  :cpp:func:`CeedOperatorAssembleLinearQFunction`) or to evaluate the diagonal
+  (:cpp:func:`CeedOperatorAssembleLinearDiagonal`).  These operations are useful for
+  preconditioning ingredients and are used in the libCEED's multigrid examples.
+* The inverse of separable operators can be obtained using
+  :cpp:func:`CeedOperatorCreateFDMElementInverse` and applied with
+  :cpp:func:`CeedOperatorApply`.  This is a useful preconditioning ingredient,
+  especially for Laplacians and related operators.
+* New functions: :cpp:func:`CeedVectorNorm`, :cpp:func:`CeedOperatorApplyAdd`,
+  :cpp:func:`CeedQFunctionView`, :cpp:func:`CeedOperatorView`.
+* Make public accessors for various attributes to facilitate writing composable code.
+* New backend: ``/cpu/self/memcheck/serial``.
+* QFunctions using variable-length array (VLA) pointer constructs can be used with CUDA
+  backends.  (Single source is coming soon for OCCA backends.)
+* Fix some missing edge cases in CUDA backend.
+
+Performance Improvements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* MAGMA backend performance optimization and non-tensor bases.
+* No-copy optimization in :cpp:func:`CeedOperatorApply`.
+
+Interface changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* Replace :code:`CeedElemRestrictionCreateIdentity` and
+  :code:`CeedElemRestrictionCreateBlocked` with more flexible
+  :cpp:func:`CeedElemRestrictionCreateStrided` and
+  :cpp:func:`CeedElemRestrictionCreateBlockedStrided`.
+* Add arguments to :cpp:func:`CeedQFunctionCreateIdentity`.
+* Replace ambiguous uses of :cpp:enum:`CeedTransposeMode` for L-vector identification
+  with :cpp:enum:`CeedInterlaceMode`.  This is now an attribute of the
+  :cpp:type:`CeedElemRestriction` (see :cpp:func:`CeedElemRestrictionCreate`) and no
+  longer passed as ``lmode`` arguments to :cpp:func:`CeedOperatorSetField` and
+  :cpp:func:`CeedElemRestrictionApply`.
+
+Examples
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+libCEED-0.6 contains greatly expanded examples with :ref:`new documentation <Examples>`.
+Notable additions include:
+
+* :file:`examples/petsc/bpsraw.c` (formerly ``bps.c``): transparent CUDA support.
+* :file:`examples/petsc/bps.c` (formerly ``bpsdmplex.c``): performance improvements and
+  transparent CUDA support.
+* :file:`examples/petsc/multigrid.c`: new p-multigrid solver with algebraic multigrid
+  coarse solve.
+* :ref:`example-petsc-bps-sphere` (:file:`examples/petsc/bpssphere.c`): generalizations
+  of all CEED BPs to the surface of the sphere; demonstrates geometric dimension
+  different from topological dimension.
```

Comment:
Do we want to mention the `area` example? It is in the docs

Reply:
Didn't seem that significant considering that full bps on the sphere were available, but I'll add if you think it's notable.

Reply:
I would say yes for completeness sake